### DISCUSSION TITLE
Using maxLength with q-editor?

### DISCUSSION THREAD

**Discussion by jammy-git** [quasarframework/quasar#8955] [General - Components / Directives / etc]
I need to limit the input, either directly or via copy and paste into a q-editor.

I understand that the editor does not have a `maxlength` property, however q-field does. But wrapping a q-editor in a q-field and setting the `maxlength` property does not seem to then limit the field input.

Am I doing something wrong, or is it correct that this does not work? If so, is there any way to achieve this requirement?

**Comment by ontwikkelfabriek** [comment]
@jammy-git you could try to put a watch on the q-editor model  and check there if the new content is too big, and if so revert the model to the old state.


```
watch: {

    myEditorModel: function (newText, oldText) {
         if(newText.length > 1000) {
            this.myEditorModel = oldText
       }
      
    }
```
